bioconda_utils/bioconductor_skeleton.py

```diff
@@ -550,6 +507,27 @@ def _parse_dependencies(self, items):
                 raise ValueError("Found {0} toks: {1}".format(len(toks), toks))
         return results
 
+    def pin_version(self, name):
+        """
+        Given a package name, return the version pin string.
+
+        For example, if version 1.2.3 is in the current bioconductor release,
+        then return >=1.2.0,<1.4.0.
```

Comment:
We could go either way on that. In practice development versions aren't released, but at least with the current set up if someone makes a package for one it'll use the at-the-time most-recent bioconductor release rather than one that hasn't come out yet. I don't really have a preference there myself, but obviously if @nturaga and the bioconductor folks would prefer a narrower pinning we can easily make that happen.

Reply:
Hi @nsoranzo , I agree with your comment. 

@dpryan79 I think I would prefer a narrower pinning because `1.4.0` honestly doesn't do anything of consequence. It is saying that you are looking for a package in the "upcoming" release, which doesn't yet exist.  Maybe @mtmorgan can add more?

Reply:
OK, we'll narrow the pinning, that's not a problem at all.